Submetrizable, equivalent definition (#1822)
pi-base/data: properties/P000112.md

The topology contains a coarser topology which is {P53}.

> Equivalently, $X$ is submetrizable when it satisfies one of the following:

Reviewer: ```suggestion
This is equivalent to each of the following:
```
Note: in general, we avoid putting the name of the property when defining it.

1. There is a continuous injective map from $X$ into a metrizable space.

> 2. There is a continuous metric on $X$.

Reviewer: ```suggestion
- There is a continuous injective map from $X$ into a metrizable space.
- There is a continuous metric on $X$.
```
This is a trivial comment.  But we usually use bullet lists.  Is there a reason for wanting to have a numbered list?

Author: @prabau I usually add numbered lists
